src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+* Added `BaggageFormat` an `ITextFormat` implementation for managing Baggage
```

Comment:
TextFormatContext is now used instead of ActivityContext in the `ITextFormat`

Reply:
just added!